lunatrace/bsl/frontend/src/api/graphql/getProject.graphql

```diff
@@ -28,10 +28,10 @@ query GetProject($project_id: uuid!) {
             created_at
             last_used
             created_by_user {
-              traits
+                traits
             }
         }
-        builds(order_by: {created_at: desc}) {
+        builds(order_by: {created_at: desc}, limit: $build_limit) {
```

Comment:
It would be cool to do offset/limit pagination here but I can see how that would get really difficult w/ RTK. People probably won't scroll down that far anyway.